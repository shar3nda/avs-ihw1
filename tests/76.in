10
10 3 2 6 -1 9 10 1 3 6