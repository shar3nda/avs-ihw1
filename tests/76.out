-1 9 10 1 3 6